build.gradle

```diff
@@ -346,16 +346,16 @@ dependencies {
   testCompile  group: 'org.springframework.statemachine', name: 'spring-statemachine-test', version: versions.springStatemachine
   testCompile 'com.github.hmcts:fortify-client:1.2.2'
   //pact contract testing
-  //contractTestCompile group: 'au.com.dius', name: 'pact-jvm-consumer-junit5_2.12', version: versions.pact
-  //contractTestCompile group: 'au.com.dius', name: 'pact-jvm-consumer-java8_2.12', version: versions.pact
-  //contractTestCompile group: 'org.hamcrest', name: 'java-hamcrest', version: '2.0.0.0'
+  contractTestImplementation group: 'au.com.dius', name: 'pact-jvm-consumer-junit5_2.12', version: versions.pact
+  contractTestImplementation group: 'au.com.dius', name: 'pact-jvm-consumer-java8_2.12', version: versions.pact
+  contractTestImplementation group: 'org.hamcrest', name: 'java-hamcrest', version: '2.0.0.0'
 
-  //contractTestCompile("org.junit.jupiter:junit-jupiter-api:5.7.0")
-  //contractTestRuntime("org.junit.jupiter:junit-jupiter-engine:5.7.0")
```

Comment:
contractTestRuntime is not supported any more. The contractTestRuntimeOnly is recommended one now.